CHANGELOG.md

```diff
@@ -1,12 +1,22 @@
 # Change Log for dash-table
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
+
 ## [Unreleased]
+### Added
+[#317](https://github.com/plotly/dash-table/issues/317)
+- New `column.selectable` nested prop that displays a selection checkbox or radio button in the column.
+- New `column_selectable` prop to choose whether columns can be selected or not, and whether a single or
+    multiple selections can be in effect at the same time.
+- New `selected_columns` prop that contains the list of visible and hidden columns that are currently selected
+- New `derived_viewport_selected_columns` that contains the list of visible columns that are currently selected
+    This prop is read-only. Use `selected_columns` in callbacks instead.
+
 ### Fixed
 [#533](https://github.com/plotly/dash-table/issues/533)
 - Fixed problem clearing one column shifting everything to the left and
 leaving the last column blank
-- Add merge_duplicate_headers prop to correct `export_format: display` behaviour.
+- Add merge_duplicate_headers prop to correct `export_format: display` behavior.
```

Comment:
heh I'm sure we have British spelling all over the place... it's important not to use it in public API but I usually let it slide in text. I won't complain about removing it though :)

Reply:
I have a spellchecker running with `en-US` as the default in vscode - it's not great but when it catches things like that I try and fix them.